graphql/secret/mutation.ts

```diff
@@ -63,24 +63,24 @@ export class MutateSecretResolver {
         @Arg('id', { nullable: true }) id?: number,
         @Arg('token', { nullable: true }) token?: string
     ) {
-        let tokenId = id;
-        let deviceId = undefined;
+        let secret = undefined;
+
         if (id) {
-            deviceId = (await prisma.secret.findUnique({ where: { id: tokenId } })).lastUsedDeviceId;
+            secret = await prisma.secret.findUnique({ where: { id } });
```

Comment:
It's a bit tricky that here we also allow retrieving secrets of other users, and only later validate in revokeSecret. Maybe we could additionally filter `userId: getSessionUser(context).userID`